setup/plugin/register.sh

```
@@ -8,7 +8,7 @@ fi
 echo "Using operator plugin to opt-in into NEAR SFFL"
 
 docker run --rm \
-  -v $(pwd):/near-sffl/ \
+  -v $(pwd):/nffl/ \
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)